circle-test.sh

```diff
@@ -59,6 +59,8 @@ exit_if_fail cd $GOPATH/src/github.com/demisto/alfred
 exit_if_fail go get -t -d -v ./...
 
 # Touch missing private files
+exit_if_fail touch $GOPATH/src/github.com/demisto/alfred/static/master/_gtmid.json
+echo '{}' >> $GOPATH/src/github.com/demisto/alfred/static/master/_gtmid.json
```

Comment:
not sure about this lines here.. just mocking the `_gtmid.json` file cause i couldn't find anything about it. Everything works as expected for me. Let me know